shared/js/textChatView.jsx

```diff
@@ -442,9 +441,6 @@ loop.shared.views.chat = (function(mozL10n) {
             dispatcher={this.props.dispatcher}
             showPlaceholder={!hasSentMessages}
             textChatEnabled={this.state.textChatEnabled} />
-          <sharedViews.AdsTileView
```

Comment:
The component should probably get an XXX statement for akita saying its currently unused & we either need to remove it or use it (maybe get a bug on file as well, as I don't think the requirement for the ToC is tracked too well).